### DISCUSSION TITLE
Infinitely flat rotation curves incompatible with Dark Matter

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#239] [Astrophysics and Physics]
https://tritonstation.com/2024/06/18/rotation-curves-still-flat-after-a-million-light-years/

**Comment by mikehelland** [comment]
I was under the impression that we now detect the Keplerian decline in the Milky Way:

https://arxiv.org/abs/2309.00048

This implies, I think, that the Milky Way has little or no dark matter.

**Reply by LifeIsStrange** [reply]
Thanks I didn't know about that! This decline seems in contradiction with the weak lensing analysis I linked (unless the length of the flat curve has extreme variations from a galaxy to another or that the milky way is a minor subpopulation that behave differently)

**Reply by budrap00** [reply]
Stacy McGaugh just addressed this [here](https://tritonstation.com/2024/06/18/rotation-curves-still-flat-after-a-million-light-years/comment-page-1/#comment-22438).

**Reply by LifeIsStrange** [reply]
We should find a definite answer to this and other fundamental questions in the next 1-3 years, based on GAIA DR4. (including the observation of a breakdown of classical gravity in wide binaries)
However, few seem to know that apparently, it is possible to match GAIA astrometry accuracy with wide field spectrographs.
Since I am a layman I don't understand _how_ spectrographs compare and allow to constrain astrometry versus GAIA,
but the WEAVE wide spectroscopic survey seems to claims parity and in some aspects superiority and complementarity to GAIA 
https://arxiv.org/abs/2212.03981
See figure 5
If so, that is excellent, because **WEAVE** is an "independent" probe with a different instrument that might have different observational biases.
Moreover combined their astrometrics results should be much more accurate (how much)
possibly better at long distances:
> complement Gaia with line-of-sight velocities and metallicities for
stars too faint for the RVS, and thereby yield the best-yet constraints
on the halo potential’s lumpiness and total mass to large distances
(∼100 kpc, e.g. Cautun et al. 2020)

 (unclear if for radial velocities too)

> This low-
resolution, high radial-velocity accuracy survey will observe ∼ 500
lines of sight through the Galactic disc at very low latitudes to probe
fundamental Galactic disc dynamics processes in a **volume inaccessi-
ble to Gaia alone.** The survey will observe 7 × 105 (five-year survey)
to 1.1×106 (seven-year survey) stars

How much more stars?
hint: https://www.prophy.science/article/173626575/

The paper outline many synergies but I haven't read it well yet.
**WEAVE** has apparently already? started and should have its first release around the same time as GAIA DR4. So IMHO this not well known survey might give us a definitive answer.
Moreover, it is unclear to me how other revolutionnary spectroscopic surveys starting very soon, compare to it for radial velocities accuracy, especially the impressive **[4MOST](https://arxiv.org/abs/1903.02464)** survey and also euclid/DESI/ZTF/roman/LSST/ANDES/HSC spectroscopy (the later sadly all ? avoid the milky way iirc? but not 4MOST).
> Where 4MOST shines is peculiar velocity science and GRoS:
i.e., mapping the large scale density and velocity fields,
enabling tests of gravity on the very largest scales (>> 10 Mpc)

Edit: there will be at least a DESI milky way survey https://www.prophy.science/article/179528592/ when?
https://arxiv.org/abs/2208.08514
> can measure radial velocities to ' 1 km s−1

> WS program will observe approxi-
mately seven million stars to a limiting magnitude of
r = 19

> low-mass stars within 100 pc of the Sun

there is also [taipan](https://arxiv.org/abs/1706.01246)
Note that taipan while having a small mirror, for some reason claims:
>  the **ultimate** peculiar velocity and bulk flow experiment, with an homogenous set of Fundamental Plane distance measurements for all ~80000 z<0.1 early type galaxies in the Taipan area;https://ui.adsabs.harvard.edu/abs/2020bugm.conf...75T/abstract
BTW makes me wonder if [cosmicflows 4](https://www.prophy.science/article/164624830/) could help

The currently (partially) released best survey for radial velocities is probably H3
https://arxiv.org/abs/1907.07684
https://arxiv.org/pdf/2406.12969
or maybe taipan
no idea about [GAMA](https://arxiv.org/abs/2203.08539)

Another thing to look at is the SphereX survey which should have main results in the second half of 2025.

BTW note that GAIA already came with a spectrograph however 1) its spectrograph is inferior and 2) has suffered heavy damage (resolution loss) by an accident.

Also since asteroseismology give the best constraints on most stellar parameters (maybe not radial velocities though), the incoming PLATO survey might have a big synergy (which is often IMO erroneously only considered for exoplanet science)
https://arxiv.org/abs/2406.05447 (section 4)
example of synergy
https://www.prophy.science/article/175893180/
